status-im/status-react
# Leftovers from desktop branch since 9th July

### Summary:

In PR #5266 desktop code was added to `develop` branch. But while merging was in progress, more commits were added to `desktop`. So they also need to be merged and after that desktop team can fully move to work in `develop`.

Current PR has squashed commits from `desktop` since 9th July, among them:
- desktop HDPI support
- selectable text in messages
- "add contact" code validation
- changed screen dimensions (1024x768)
- navigating to profile from user chat
- commit-sha in desktop version
- correct folder for keystore and realm files



### Review notes (optional):


### Testing notes (optional):


### Steps to test:
- Open Status
- Check if it works :)



status: ready
